tests/smoke/test.cli.build.js

```diff
@@ -0,0 +1,32 @@
+/* @flow */
+import {spawn} from 'child_process';
+import {describe, it} from 'mocha';
+
+import {
+  webExt, addonPath, artifactsPath,
+} from './common';
+
+describe('web-ext build', () => {
+  it('webext build --source-dir SRCDIR --artifacts-dir OUTDIR', () => {
+    return new Promise((resolve, reject) => {
+      const webextProcess = spawn(webExt, [
+        'build',
+        '--source-dir', addonPath,
+        '--artifacts-dir', artifactsPath,
```

Comment:
I know it's a bit more effort, but I would rather see `artifactsPath` done with a temp directory that gets torn down after the test (preferably also with an at-exit handler to guard against exceptions). You could maybe just use [TempDir](https://github.com/mozilla/web-ext/blob/0f87c9b986178db6d4e6a3e2d880cf2dfd7e873b/src/util/temp-dir.js#L51) actually. I'm worried that there could be some data leakage between tests if we rely on the project level artifact directory for this.